.gitignore

```diff
@@ -15,7 +15,6 @@
 /tmp
 
 config/database.yml
-config/site.yml
```

Comment:
Sorry I didn't get you, can you please explain a little more ?

In this their is application.yml file created in which we can store our secrets and which will not be pushed to github, the changes made in the site.yml file will be pushed to github.